templates/default/src/utils/apollo.jsx

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable react/jsx-props-no-spreading, no-console */
+/* FOR SERVER SIDE RENDERING APOLLO FETCHES: Use unmodifed link below.
+ * https://github.com/zeit/next.js/blob/canary/examples/with-apollo/lib/apollo.js
+ * this file is modified from link above. Had some issues getting it to work with hooks.
+ */
+import React from 'react'
+import { ApolloProvider } from '@apollo/react-hooks'
+import { ApolloClient } from 'apollo-client'
+import { InMemoryCache } from 'apollo-cache-inmemory'
+import { HttpLink } from 'apollo-link-http'
+import fetch from 'isomorphic-unfetch'
```

Comment:
Do we need the fetch polyfill? Are we expecting to support IE11?
https://caniuse.com/#feat=fetch

Reply:
yes we are expecting to support IE11 as based on HVT